dunwoody_disaster/views/MapScreen.py

```diff
@@ -2,16 +2,18 @@
 The entry point for the game
 """
 
-import sys
 from PySide6.QtCore import Qt
-from PySide6.QtWidgets import QApplication, QMainWindow, QGridLayout, QWidget, QLabel
+from PySide6.QtWidgets import QGridLayout, QWidget, QLabel
 from PySide6.QtGui import QPixmap, QKeyEvent
 from dunwoody_disaster import ASSETS
 
 
+
+
 class MapScreen(QWidget):
-    def __init__(self):
+    def __init__(self, callback):
```

Comment:
this should have a type hint of `Callable` which you will need to `from typing import Callable`. Also, can we come up with a more descriptive name? A docstring would also be helpful so we can understand what the point of this parameter is.

Reply:
I can add a docstring, but I don't really know what it's doing tbh.

Reply:
Maybe looking at [#36](https://github.com/chanson02/DunwoodyDisaster/pull/36/files/1aff6571c13710d822c2c67cf9795df9552c5bcc..ce3d3eaea8de44863695a320485551761daa83dd) would help you. On the `VictoryScreen` there's a function `loot_selected_event` which is passed to the `LootScreen`. Once the player has selected their loot, the loot screen will call that event function on the victory screen.